packages/core/src/button/base-button.element.scss

```diff
@@ -105,8 +98,13 @@
 ::slotted(a) {
   text-decoration: none !important;
   display: block;
-  height: 100%;
   color: inherit;
+  margin: calc(var(--padding) * -1);
+  padding: var(--padding);
+  min-width: var(--min-width);
+  line-height: 1em;
+  box-sizing: border-box !important;
+  outline: 0;
```

Comment:
Fixes anchors to match the same size as the button so the touch target and focus styles are correct.